app/src/sharedTest/java/org/oppia/android/app/options/OptionsFragmentTest.kt

```diff
@@ -485,6 +545,10 @@ class OptionsFragmentTest {
       ).perform(click())
       intended(
         allOf(
+          hasExtra(
+            AudioLanguageActivity.KEY_AUDIO_LANGUAGE_PREFERENCE_TITLE,
+            AUDIO_LANGUAGE
+          ),
```

Comment:
Add the below check in `openOptionsActivity_configChange_clickAudioLanguage_opensAudioLanguageActivity` test as well
```kt
hasExtra(
            AudioLanguageActivity.KEY_AUDIO_LANGUAGE_PREFERENCE_TITLE,
            AUDIO_LANGUAGE
          ),
```

Reply:
Yea, I added it back.